Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -715,7 +715,7 @@ public function fulltext($searchWord)
     {
         $this->appendAtPath('query.filtered.query.bool.must', array(
             'query_string' => array(
-                'query' => $searchWord
+                'query' => str_replace('/', '\\/', $searchWord)
```

Comment:
hmm maybe it's actually needed to pass it through `json_encode` instead? this seems hacky at best